static/app/views/issueList/savedSearchTab.tsx

```diff
@@ -35,111 +42,179 @@ function SavedSearchTab({
+      });
+    }
+
+    return searchOptions;
+  }, []);
+
+  const trigger = ({props, ref}) => (
+    <StyledDropdownTrigger
+      ref={ref}
+      {...props}
+      isActive={isActive}
+      borderless
+      priority="link"
+      size="zero"
+    >
       {isActive ? (
         <Fragment>
-          <TitleTextOverflow>
-            {savedSearch ? savedSearch.name : t('Custom Search')}{' '}
-          </TitleTextOverflow>
-          {queryCount !== undefined && queryCount > 0 && (
-            <div>
-              <Badge>
-                <QueryCount hideParens count={queryCount} max={1000} />
-              </Badge>
-            </div>
+          <span>{savedSearch ? savedSearch.name : t('Custom Search')}&nbsp;</span>
+          {defined(queryCount) && queryCount > 0 && (
+            <Badge>
+              <QueryCount hideParens count={queryCount} max={1000} />
+            </Badge>
           )}
         </Fragment>
       ) : (
         t('Saved Searches')
       )}
-    </TitleWrapper>
+    </StyledDropdownTrigger>
+  );
+
+  const onChange = useCallback(
+    option => {
+      const searchObj = savedSearchList.find(s =>
+        s.isGlobal
+          ? `global-search-${s.id}` === option.value
+          : `saved-search-${s.id}` === option.value
+      );
+      searchObj && onSavedSearchSelect(searchObj);
+    },
+    [savedSearchList, onSavedSearchSelect]
   );
 
   return (
-    <TabWrapper
+    <StyledCompactSelect
+      renderWrapAs="li"
+      trigger={trigger}
+      options={options}
+      value={savedSearchValue}
       isActive={isActive}
-      className="saved-search-tab"
-      data-test-id="saved-search-tab"
-    >
-      <StyledDropdownLink
-        alwaysRenderMenu={false}
-        anchorMiddle
-        caret
-        title={title}
-        isActive={isActive}
-      >
-        <SavedSearchMenu
-          organization={organization}
-          savedSearchList={savedSearchList}
-          onSavedSearchSelect={onSavedSearchSelect}
-          onSavedSearchDelete={onSavedSearchDelete}
-          query={query}
-          sort={sort}
-        />
-      </StyledDropdownLink>
-    </TabWrapper>
+      onChange={onChange}
+      offset={-4}
+      maxMenuHeight={800}
+    />
   );
 }
 
 export default SavedSearchTab;
 
-const TabWrapper = styled('li')<{isActive?: boolean}>`
-  /* Color matches nav-tabs - overwritten using dark mode class saved-search-tab */
-  border-bottom: ${p => (p.isActive ? `4px solid #6c5fc7` : 0)};
-  /* Reposition menu under caret */
-  & > span {
-    display: block;
-  }
-  & > span > .dropdown-menu {
-    padding: 0;
-    margin-top: ${space(1)};
-    min-width: 20vw;
-    max-width: 25vw;
-    z-index: ${p => p.theme.zIndex.globalSelectionHeader};
-
-    :after {
-      border-bottom-color: ${p => p.theme.backgroundSecondary};
-    }
+const StyledCompactSelect = styled(CompactSelect)<{isActive?: boolean}>`
+  && {
```

Comment:
what is `&&`

Reply:
oh it's just a double selector to override the navigation tab styles:
<img width="498" alt="Screen Shot 2022-05-04 at 1 05 01 PM" src="https://user-images.githubusercontent.com/44172267/166816919-b5385d33-c10f-471d-b6c1-c890c39ee289.png">